sql/src/test/java/io/crate/integrationtests/GroupByAggregateTest.java

```diff
@@ -1239,8 +1239,9 @@ public void testAggregateOnIndexOff() throws Exception {
 
     @Test
     public void testGroupBySingleLongWithNullValues() {
-        execute("create table t (x long) clustered into 2 shards with (number_of_replicas = 0)");
-        execute("insert into t (x) values (1), (1), (1), (2), (2), (null), (null), (null), (null)");
+        execute("create table t (x long, y int, z short) clustered into 2 shards with (number_of_replicas = 0)");
```

Comment:
Maybe include the type in the column name? I know it's generally not the best practice, but I think for the purpose of the test it would help with readability.